### ISSUE TITLE
Determine the AI model we need to use

### ISSUE THREAD

**Issue by samduwu** [samduwu/nonogram-ai#4]
We determined that OCR is likely the one, but should verify after more research.

Then,
* Install PyTorch
* Update the requirements.txt file
* Create a Jupyter Notebook to hold preliminary code (this will be pushed to this repo)
* Determine how to create said model in PyTorch and determine the best way to calculate loss
* If code is written for the above, figure out how to intake images as training data (determine how the images should be labeled, if necessary)
* Push the notebook to the repo
